CONTRIBUTING.md

```diff
@@ -0,0 +1,32 @@
+## Contributing Issues
+
+### Before Submitting an Issue
+First, if your issue is a question or you need help with setting up / configuring Streisand, then please ask on the [streisand-discussions](https://github.com/jlund/streisand-discussions/) repo.
+
+If a bug or technical feature request, please do a search in [open issues](https://github.com/jlund/streisand/issues) to see if the issue or feature request has already been filed.
+
+If you find your issue already exists, make relevant comments and add your [reaction](https://github.com/blog/2119-add-reactions-to-pull-requests-issues-and-comments). Use a reaction in place of a "+1" comment.
+
+👍 - upvote
+
+👎 - downvote
+
+If you cannot find an existing issue that describes your bug or feature, submit an issue using the guidelines below.
+
+## Writing Good Bug Reports and Feature Requests
+
+File a single issue per problem and feature request. Do not include multiple bugs or feature requests in the same issue.
+The more information you can provide, the more likely someone will be successful reproducing the issue and finding a fix. 
+
+#### Please include the following with each issue:

```

Comment:
Can you add a bullet for log output? Ideally mentioning using a Gist or surrounding the output with backticks (lots of people will paste raw logs into the comment box otherwise!)